addons/base_setup/static/src/xml/res_config_invite_users.xml

```diff
@@ -1,20 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates>
-    <div t-name='res_config_invite_users'>
+    <div t-name='res_config_invite_users' owl="1">
         <p class="o_form_label">Invite New Users</p>
+
         <div class="d-flex">
-            <input class="o_user_emails o_input mt8" type="text" placeholder="Enter e-mail address"/>
-            <button class="btn btn-primary o_web_settings_invite" data-loading-text="Inviting..."><strong>Invite</strong></button>
+            <input t-model="state.emails" t-att-disabled="state.status != 'idle'" class="o_user_emails o_input mt8" t-on-keydown="(ev) => this._onKeydownUserEmails(ev)" type="text" placeholder="Enter e-mail address"/>
```

Comment:
t-on-keydown="(ev) => this._onKeydownUserEmails(ev)" -> t-on-keydown="_onKeydownUserEmails"

Reply:
several handlers can be "simplified" like this